src/server/server.rs

```diff
@@ -48,9 +50,11 @@ pub struct Server<T: RaftStoreRouter + 'static, S: StoreAddrResolver + 'static>
     // For sending/receiving snapshots.
     snap_mgr: SnapManager,
     snap_worker: Worker<SnapTask>,
+    debugger: Option<Worker<DebugTask>>,
 }
 
 impl<T: RaftStoreRouter, S: StoreAddrResolver + 'static> Server<T, S> {
+    #[allow(too_many_arguments)]
```

Comment:
IMO, I don't like `allow(too_many_arguments)` and we should refactor all in our current projects too.

Reply:
/cc @BusyJay 

We should make the code more elegant later.

Reply:
Sure! I will refator it in another PR.